### ISSUE TITLE
SQL Aliases are converted to lower case

### ISSUE THREAD

**Issue by tomdottom** [liquidata-inc/dolt#541]
For example the `positiveIncrease` alias in this query:

```sql
SELECT
  date,
  state,
  positiveIncrease AS positiveIncrease,
  LPAD ('', positiveIncrease / (SELECT (MAX(positiveIncrease)/25) FROM states_daily WHERE state = 'NY'), '#') AS graph
FROM states_daily
WHERE state = 'NY'
AND positiveIncrease IS NOT NULL
ORDER BY date DESC
;
```

Is converted to lowercase in the output table.  It is also lowercase when using the csv output formatter, and in the web query interface.

```
+-------------------------------+-------+------------------+---------------------------+
| date                          | state | positiveincrease | graph                     |
+-------------------------------+-------+------------------+---------------------------+
| 2020-04-08 00:00:00 +0000 UTC | NY    | 10453            | ########################  |
| 2020-04-07 00:00:00 +0000 UTC | NY    | 8174             | ##################        |
| 2020-04-06 00:00:00 +0000 UTC | NY    | 8658             | ###################       |
| 2020-04-05 00:00:00 +0000 UTC | NY    | 8327             | ###################       |
| 2020-04-04 00:00:00 +0000 UTC | NY    | 10841            | ######################### |
| 2020-04-03 00:00:00 +0000 UTC | NY    | 10482            | ########################  |
| 2020-04-02 00:00:00 +0000 UTC | NY    | 8669             | ####################      |
| 2020-04-01 00:00:00 +0000 UTC | NY    | 7917             | ##################        |
| 2020-03-31 00:00:00 +0000 UTC | NY    | 9298             | #####################     |
| 2020-03-30 00:00:00 +0000 UTC | NY    | 6984             | ################          |
| 2020-03-29 00:00:00 +0000 UTC | NY    | 7195             | ################          |
| 2020-03-28 00:00:00 +0000 UTC | NY    | 7683             | #################         |
| 2020-03-27 00:00:00 +0000 UTC | NY    | 7377             | #################         |
...
```

https://www.dolthub.com/repositories/tomdottom/covid-tracking/query/master?q=SELECT%0A%20%20date%2C%0A%20%20state%2C%0A%20%20positiveIncrease%20AS%20positiveIncrease%2C%0A%20%20LPAD%20(%27%27%2C%20positiveIncrease%20%2F%20(SELECT%20(MAX(positiveIncrease)%2F25)%20FROM%20states_daily%20WHERE%20state%20%3D%20%27NY%27)%2C%20%27%23%27)%20AS%20graph%0AFROM%20states_daily%0AWHERE%20state%20%3D%20%27NY%27%0AAND%20positiveIncrease%20IS%20NOT%20NULL%0AORDER%20BY%20date%20DESC

**Comment by timsehn** [comment]
I've definitely seen this. I thought there was a skipped bats test for it.
I'll dig around and see if I can find it. Otherwise, I'll create the bats
tests. Fixing skipped bats tests is a priority for Andy on our team.

**Comment by timsehn** [comment]
Here's the skipped test. It's in the backlog.

https://github.com/liquidata-inc/dolt/blob/master/bats/capital-letter-column-names.bats#L60

**Comment by zachmu** [comment]
Hi Tom,

We're tracking this for the next release (Monday). Stay tuned.

**Comment by zachmu** [comment]
Fixed in 0.16.0
